src/lib/boavizta.ts

```diff
@@ -1,7 +1,17 @@
-import {IImpactModelInterface} from './interfaces/index';
 import axios from 'axios';
 
-export {IImpactModelInterface} from './interfaces/index';
```

Comment:
@narekhovhannisyan This is essential for JSII to build.

Reply:
[jsii.patch](https://github.com/Green-Software-Foundation/ief/files/12389418/jsii.patch)

Patch File for your reference. Few more changes are also aggregated here.

Reply:
Great, meanwhile checking build directory issue